src/actions/marketo/marketo.ts

```diff
@@ -31,10 +32,16 @@ export class MarketoAction extends Hub.Action {
 
   ]
   supportedActionTypes = [Hub.ActionType.Query]
-  supportedFormats = [Hub.ActionFormat.JsonDetail]
   supportedFormattings = [Hub.ActionFormatting.Unformatted]
   supportedVisualizationFormattings = [Hub.ActionVisualizationFormatting.Noapply]
   usesStreaming = true
+  supportedFormats = (request: Hub.ActionRequest) => {
```

Comment:
I would write a quick test for these two actions that validates the behavior on different looker versions

Reply:
good idea